src/tagmanager/tm_tag.c

```diff
@@ -82,7 +82,11 @@ typedef struct
 	gboolean first;
 } TMSortOptions;
 
-/* Gets the GType for a TMTag */
+/** Gets the GBoxed-derived GType for a TMTag.
```

Comment:
I followed the same procedure as for TMSourceFile where this was accepted. I'm telling it's a GBoxed so that you you know can use g_boxed_copy/free, since the ref/unref functions are not exported.

I have no problem with exporting ref/unref (and maybe removing the GBoxed reference from the comment) if you prefer.